jwst/exp_to_source/tests/test_main.py

```diff
@@ -15,7 +15,7 @@ def test_default_run(tmp_path, capsys):
         '-o',
         str(tmp_path)
     ]
-    args.extend(helpers.INPUT_FILES)
+    args.extend([str(infile) for infile in helpers.INPUT_FILES])
```

Comment:
why was this change necessary? (here and below). This looks like an indication that `helpers.INPUT_FILES` may have changed its behavior

Reply:
yes, this was necessary because pathlib returns an object rather than a list.

Reply:
That makes sense, but IMO the conversion back to string should live inside `helpers.INPUT_FILES`, i.e., the behavior of that constant doesn't change

Reply:
agreed, that is a better place to code the change.